cloudify_agent/installer/__init__.py

```diff
@@ -387,9 +391,30 @@ def upload_certificate(self):
         cert_content = self.rest_cert_content
 
         if not self.runner.exists(cert_file):
+            cert_dir = os.path.dirname(cert_file)
```

Comment:
if the agent is a windows agent, will running os.path.dirname on the linux manager on certfile do the right thing?